lib/groonga/command/parser/command/groonga-command-convert-format.rb

```diff
@@ -78,6 +79,17 @@ def parse_options!(argv)
               @pretty_print = boolean
             end
 
+            option_parser.on("--elasticsearch-version=VERSION",
+                             "Specify the Elasticsearch version",
+                             "Because the Elasticsearch's importformat" +
```

Comment:
"import format"? Is "importformat" (no whitespace) a term in Elasticsearch?

Reply:
Ah...This is a typo.
"importformat" isn't a term in Elasticsearch.